app/assets/stylesheets/widgets.scss

```diff
@@ -365,3 +365,9 @@
     }
   }
 }
+
+.sponsorship-dropdown {
+  hr {
+    border: 1px solid var(--divider);
+  }
```

Comment:
Also, as per mocks the left-right margins of the divider from the entire box-border is 16px, out of which 8 has already been accounted for in current UI, so we should add margin left-right `8px` to this `hr`.

Reply:
Again, good catch, thank you!